src/main/fc/runtime_config.c

```diff
@@ -23,6 +23,8 @@
 #include "fc/runtime_config.h"
 #include "io/beeper.h"
 
+const char *armingDisableFlagNames[] = { ARMING_DISBALED_FLAG_NAMES };
```

Comment:
Ah, I thought you were referring to the process of getting hay ready to be fed to cattle with `DISBALE`. :-P